pyiron_contrib/__init__.py

```diff
@@ -1,7 +1,11 @@
 __version__ = "0.1"
 __all__ = []
 
-from pyiron import Project
+try:
+        from pyiron import Project
+except:
+        print("Warning: pyiron module not found;\nBe carefule not to use \"from pyiron_contrib import Projects\"")
```

Comment:
Please do not use print but instead warnings.warn() in addition we should import the project from pyiron_base.

Reply:
Done!
https://github.com/pyiron/pyiron_contrib/blob/f51c4a6991afa1511a36bfc2e6ae5b0ad861ed86/pyiron_contrib/__init__.py#L9